app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -55,48 +55,67 @@ keywords:
   - Settlement Scheme
   
 ---
+If you are applying for teacher training as a non-UK citizen, you will need a visa or immigration status allowing you to study or work in England. 
 
-## How to apply for your visa
+If you do not already have a visa or immigration status allowing you to study or work in England, you will need to apply for a Student or Skilled Worker visa. 
```

Comment:
```suggestion
If you do not already have a visa or immigration status allowing you to study or work in England, you will need to apply for a Student visa or Skilled Worker visa. 
```
Is it worth tweaking this slightly to be clear these are 2 visas rather than a combined one?